src/main/java/homer/model/environment/HomeEnvironment.java

```diff
@@ -9,14 +11,16 @@
 public final class HomeEnvironment implements Environment {
 
     private Temperature temperature;
+    private AirQualityState airQualityState;
 
     /**
      * Creates a new {@link HomeEnvironment}.
      * 
      * @param temperature The initial temperature of the environment.
      */
-    public HomeEnvironment(final Temperature temperature) {
+    public HomeEnvironment(final Temperature temperature, final AirQualityState airQualityState) {
```

Comment:
Can Optionals be used here? Or you intend to always set this? I saw that you set it to null in test classes, is it what you intend to do in the future or you're going to change it in the future?

Reply:
i'm passing null just to make it compile fast, it should always have both fields set in final implementation